cli/integration_tests/basic_monorepo/dry_run_json.t

```diff
@@ -72,7 +72,8 @@ Setup
     },
     "expandedInputs": {
       "package.json": "f2a5d2525f3996a57680180a7cd9ad7310e4dec0"
-    }
+    },
+    "framework": "<FRAMEWORK NOT DETECTED>"
```

Comment:
If we get #4000 merged, there will be a nice opportunity to expand our test suite to use existing `examples/`. Then we don't need to create a new test fixture for integration tests that includes, say, Next.js as a dependency.